tests/unit/mathematics/linalg/LinalgBackendViennaCL_unittest.cc

```diff
@@ -5,7 +5,7 @@
 #include <thread>
 
 #ifdef HAVE_VIENNACL
-#include <shogun/mathematics/linalg/LinalgBackendViennaCL.h>
+#include <shogun/mathematics/linalg/LinalgBackendViennacl.h>
```

Comment:
@lambday no hurry... prefer the camelized notation over that weird mixture of camelized and not...